### PR TITLE
Fix hpa-use-rest-clients help text

### PR DESCRIPTION
**Pull request by DirectXMan12**
The help text erroneously says "WARNING: alpha feature" when it
shouldn't have.  When we moved to beta, this should have been removed.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by DirectXMan12** [comment]
cc @kubernetes/sig-autoscaling-bugs :-/

**Comment by DirectXMan12** [comment]
We're going to need to cherry-pick this back to 1.10 and 1.9 as well.

File: cmd/kube-controller-manager/app/options/options.go [hunk 1]
```diff
@@ -136,7 +136,7 @@ func (s *KubeControllerManagerOptions) AddFlags(fs *pflag.FlagSet, allController
...
-	fs.BoolVar(&s.Generic.ComponentConfig.HorizontalPodAutoscalerUseRESTClients, "horizontal-pod-autoscaler-use-rest-clients", s.Generic.ComponentConfig.HorizontalPodAutoscalerUseRESTClients, "WARNING: alpha feature.  If set to true, causes the horizontal pod autoscaler controller to use REST clients through the kube-aggregator, instead of using the legacy metrics client through the API server proxy.  This is required for custom metrics support in the horizontal pod autoscaler.")
+	fs.BoolVar(&s.Generic.ComponentConfig.HorizontalPodAutoscalerUseRESTClients, "horizontal-pod-autoscaler-use-rest-clients", s.Generic.ComponentConfig.HorizontalPodAutoscalerUseRESTClients, "If set to true, causes the horizontal pod autoscaler controller to use REST clients through the kube-aggregator, instead of using the legacy metrics client through the API server proxy.  This is required for custom metrics support in the horizontal pod autoscaler.")
```
Comment (aleksandra-malinowska) [inline cmd/kube-controller-manager/app/options/options.go]: I'm not sure what's the convention here, but the flag above has a beta feature warning. Does it need to be added in this case as well?

**Comment by DirectXMan12** [comment]
@deads2k or someone with permissions on that directory, PTAL

**Pull request merged**